### PR TITLE
Make ready for split-controller testing in ansible-core

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Uses ansible/ansible#75605 instead of devel in CI. Let's see what happens and if there are surprises :-)

##### ISSUE TYPE
- CI Pull Request

##### COMPONENT NAME
CI

### CODE REVIEW

**Comment by felixfontein** [comment]
Restarting CI.

**Comment by felixfontein** [comment]
Restarting CI.

File: tests/integration/targets/ansible_galaxy_install/aliases [hunk 1]
```diff
@@ -1,3 +1,4 @@
 destructive
 shippable/posix/group3
 skip/python2.6
+context/controller  # While this is not really true, this module mainly is run on the controller, *and* needs access to the ansible-galaxy CLI tool
```
Comment (mattclay) [inline tests/integration/targets/ansible_galaxy_install/aliases:4]: Although this test is for a module, the module can only run on the controller, so I'd rephrase this comment:

```suggestion
context/controller  # this module only runs on the controller since it needs access to the ansible-galaxy CLI tool
```

File: tests/integration/targets/callback_diy/aliases [hunk 2]
```diff
@@ -1,2 +1,3 @@
 shippable/posix/group3
 needs/target/callback
+context/controller
```
Comment (mattclay) [inline tests/integration/targets/callback_diy/aliases:3]: Plugin tests shouldn't need this alias since `ansible-test` auto-detects them as controller tests.

File: tests/integration/targets/ansible_galaxy_install/aliases (hunk 1 above)
Comment (felixfontein) [reply tests/integration/targets/ansible_galaxy_install/aliases:4]: It can also run on the target - assuming ansible-core/ansible-base/Ansible 2.9 has been installed there.

**Comment by felixfontein** [comment]
Manual backport to stable-2 in #3113 3413.
